server.js

```diff
@@ -7606,6 +7606,7 @@ cache({
     var badgeData = getBadgeData('custom badge', query);
 
     if (!query.uri){
+      badgeData.colorB = '#e05d44'; // red
       badgeData.text[1] = 'no uri specified';
```

Comment:
Another option you could consider here is the setBadgeColor helper in badge-data.js. It will let you use named or hex colors interchangeably, discarding colorB if needed.

Reply:
That is the exact function i needed!
Did not realise it also discarded the user override.